### ISSUE TITLE
Error pushing a project

### ISSUE THREAD

**Issue by Seradjor** [chrisrhymes/bulma-clean-theme#154]
Hi,

Good morning,

I have push a new commit after a few weeks to a project with bulma theme and I have had the next error:

Error:  Logging at level: debug Configuration file: /github/workspace/./docs/_config.yml GitHub Pages: github-pages v231 GitHub Pages: jekyll v3.9.5 Theme: jekyll-theme-primer Theme source: /usr/local/bundle/gems/jekyll-theme-primer-0.6.0 Requiring: jekyll-github-metadata Requiring: jekyll-seo-tag Requiring: jekyll-coffeescript Requiring: jekyll-commonmark-ghpages Requiring: jekyll-gist Requiring: jekyll-github-metadata Requiring: jekyll-paginate Requiring: jekyll-relative-links Requiring: jekyll-optional-front-matter Requiring: jekyll-readme-index Requiring: jekyll-default-layout Requiring: jekyll-titles-from-headings Requiring: jekyll-remote-theme GitHub Metadata: Initializing... Source: /github/workspace/./docs Destination: /github/workspace/./docs/_site Incremental build: disabled. Enable with --incremental Generating... Theme: chrisrhymes/bulma-clean-theme Theme source: /tmp/jekyll-remote-theme-20240513-8-kc2wmp Remote Theme: Using theme chrisrhymes/bulma-clean-theme Remote Theme: Downloading https://codeload.github.com/chrisrhymes/bulma-clean-theme/zip/HEAD to /tmp/jekyll-remote-theme-20240513-8-1gsya0p.zip Remote Theme: Unzipping /tmp/jekyll-remote-theme-20240513-8-1gsya0p.zip to /tmp/jekyll-remote-theme-20240513-8-kc2wmp Requiring: jekyll-feed Requiring: jekyll-sitemap Requiring: jekyll-paginate Requiring: jekyll-seo-tag EntryFilter: excluded /Gemfile.lock EntryFilter: excluded /Gemfile Reading: _posts/2024-03-15-welcome-to-jekyll.markdown Theme: Ignoring assets/css/app.scss in theme due to existing file with that path in site. Generating: JekyllOptionalFrontMatter::Generator finished in 3.1048e-05 seconds. Generating: JekyllReadmeIndex::Generator finished in 4.1658e-05 seconds. Generating: Jekyll::Paginate::Pagination finished in 3.827e-06 seconds. Generating: JekyllRelativeLinks::Generator finished in 0.002704755 seconds. Generating: JekyllDefaultLayout::Generator finished in 0.000257771 seconds. Generating: JekyllTitlesFromHeadings::Generator finished in 1.7473e-05 seconds. Rendering: _posts/2024-03-15-welcome-to-jekyll.markdown Pre-Render Hooks: _posts/2024-03-15-welcome-to-jekyll.markdown Rendering Liquid: _posts/2024-03-15-welcome-to-jekyll.markdown Requiring: rouge Requiring: kramdown-parser-gfm Rendering Markup: _posts/2024-03-15-welcome-to-jekyll.markdown Requiring: kramdown-parser-gfm Rendering Layout: _posts/2024-03-15-welcome-to-jekyll.markdown Layout source: theme Rendering: _posts/2024-03-15-welcome-to-jekyll.markdown/#excerpt Pre-Render Hooks: _posts/2024-03-15-welcome-to-jekyll.markdown/#excerpt Rendering Markup: _posts/2024-03-15-welcome-to-jekyll.markdown/#excerpt Rendering: 404.html Pre-Render Hooks: 404.html Rendering Markup: 404.html Rendering Layout: 404.html Layout source: theme Rendering: about.md Pre-Render Hooks: about.md Rendering Markup: about.md Rendering Layout: about.md Layout source: theme Rendering: assets/css/app.scss Pre-Render Hooks: assets/css/app.scss Rendering Markup: assets/css/app.scss github-pages 231 | Error: Invalid CSS after "... arrow($color: ": expected expression (e.g. 1px, bold), was "#{cv.getVar("li..." on line 4 Remote Theme: Cleaning up /tmp/jekyll-remote-theme-20240513-8-kc2wmp

I have donne only data changes and I don't understand what could happend.

I have seen that you have made some updates. Could be it the reason?

Thank you.

Sincerely,

Sergio

**Comment by juliowaissman** [comment]
Hi @Seradjor,

The solution i found is to modify the theme in `_config.yml` by:

`remote_theme: chrisrhymes/bulma-clean-theme@v0.13.3`

I think that [this documentation](http://www.csrhymes.com/bulma-clean-theme/docs/getting-started/installation/#v0x-of-bulma-clean-theme) needs to be corrected.

**Comment by donkirkby** [comment]
@chrisrhymes, I can confirm the same problem with my sites. I swear they were working after my testing last week, but not now.

I fixed my sites by converting `_config.yml` to use this:

```
remote_theme: chrisrhymes/bulma-clean-theme@v0.14.0
```

I looked at [`jekyll-remote-theme`], and their examples use `@`, so I guess that's the right way to do it. I'm not sure what changed between last week and now.

[`jekyll-remote-theme`]: https://github.com/benbalter/jekyll-remote-theme?tab=readme-ov-file#declaring-your-theme
